systemd.yaml

```diff
@@ -558,8 +558,8 @@ subpackages:
             systemd-firstboot --version
             systemd-firstboot --help
 
-  - name: "systemd-repart-rootfs"
-    description: "Config file to auto grow root filesystem"
+  - name: "systemd-repart-rootfs-xfs"
```

Comment:
Non-blocking, but we could use a range here if we plan to support other filesystems outside of ext4 and xfs

Reply:
Right! Done that now, thanks :)